pkgs/development/tools/analysis/stylelint/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildNpmPackage, fetchFromGitHub, lib, ... }:
+
+buildNpmPackage rec {
+  pname = "stylelint";
+  version = "15.10.3";
+
+  src = fetchFromGitHub {
+    owner = "stylelint";
+    repo = "stylelint";
+    rev = version;
+    hash = "sha256-k7Ngbd4Z3/JjCK6taynIiNCDTKfqGRrjfR0ePyRFY4w=";
+  };
+
+  npmDepsHash = "sha256-tVDhaDeUKzuyJU5ABSOeYgS56BDSJTfjBZdTsuL/7tA=";
+
+  dontNpmBuild = true;
+
+  meta = with lib; {
+    description = "CSS linter";
+    homepage = "https://stylelint.io";
+    license = licenses.mit;
+    maintainers = with maintainers; [ SuperSandro2000 ];
```

Comment:
Since I just need to lookup what stylelint is, I don't think I should be written here.

Reply:
You authored the initial PR (https://github.com/NixOS/nixpkgs/pull/105192) so I figured you might be an appropriate maintainer, hence my requesting you for a review. Maybe @alypeng?

Reply:
I think I just fixed and closed #99113


Anyway, I don't really want to be maintainer of this and the linked person is inactive on github, so also not a good match.

Reply:
Since we are converting things away from the unmaintainable nodePackages, we could also just keep this empty for now.